### ISSUE TITLE
Disk image containing status info

### ISSUE THREAD

**Issue by tin-nl** [atarijookie/ce-atari#45]
Esp. the SCSI version currently often only gets connected via Floppy interface - many people lack the SCSI cable and/or are curious while waiting for one. Often they don't know they can connect the CE to the Network and if they know and/or do, they can't figure out the IP address.
It would be very helpfull in those cases, if drive A would contain a readme.txt that Contains CE info like IP address, SCSI/ACSI status (connected or not). etc.
Main question is, when to insert such an image - doesn't the CE currently insert an empty image at boot?

**Comment by atarijookie** [comment]
> Main question is, when to insert such an image - doesn't the CE currently insert an empty image at boot?

Yes, it inserts an empty image... 

But why would you want to find out the IP of your device from that status file? 
When I want to find the IP of my device (which gets assigned from DHCP), I either look in the client list of my router (through web interface), or use Angry IP scanner to scan the whole network. 
I understand that looking into a single file would be simpler though...

**Comment by tin-nl** [comment]
> When I want to find the IP of my device (which gets assigned from DHCP), I either look in the client list of my router (through web interface), or use Angry IP scanner to scan the whole network. 

Of course, we do it like that, but as I said, many users don't know how. They often even don't know that the ST can't configure the CE w/o ACSI connection.

That way the "I don't read/understand FAQs" crowd would at least stumble upon the webinterface. So a "welcome" disk image would go a long way for trial-and-error users.

**Comment by atarijookie** [comment]
> They often even don't know that the ST can't configure the CE w/o ACSI connection

Hmm, once there will be write support on the floppy interface, you theoretically could configure the CE even through FDD without ACSI ;) But you're right, let's help those people to get started.

**Comment by miniupnp** [comment]
why not inserting the ce_conf.st image at boot ?
it is possible to add a status file in it.
